compiler/compiler/src/test.rs

```diff
@@ -40,7 +40,16 @@ fn new_compiler(handler: &Handler, main_file_path: PathBuf) -> Compiler<'_> {
     let output_dir = PathBuf::from("/tmp/output/");
     fs::create_dir_all(output_dir.clone()).unwrap();
 
-    Compiler::new(handler, main_file_path, output_dir)
+    Compiler::new(
+        handler,
+        main_file_path,
+        output_dir,
+        Some(OutputOptions {
```

Comment:
Why do we output the ast's by default?
At the moment I think they should be only output for acl2 and testing.